draft-ietf-webtrans-overview.md

```diff
@@ -367,10 +367,10 @@ receive side aborted
 : Indicates that the peer has aborted the corresponding send side of the
   stream.  An unsigned 8-bit error code from the peer may be available.
 
-Data Recvd state reached
-: Indicates that no further data will be transmitted or retransmitted on the
-  local send side, and that the FIN has been sent.  Data Recvd implies that
-  aborting send-side is a no-op.
+all data committed
+: Indicates that all of the outgoing data on the stream, including the FIN, is
+  in the state where aborting the send side would have no further effect on any
+  data being delivered.
```

Comment:
This is a fine definition, but I think that you need to add a section explaining what that means, concretely, for different protocols.  That will necessarily involve some ambiguity and talking about layering and APIs, but this is not really very clear without that elaboration.

Reply:
Does this look better?